src/roles/common/tasks/vsc-apply-config.yml

```diff
@@ -1,3 +1,12 @@
+- name: Set VSC Creds if not defined
+  set_fact:
+    vsc_creds:
+      host: "{{ mgmt_ip }}"
+      username: "{{ vsc_username|default(vsc_default_username) }}"
+      password: "{{ vsc_password|default(vsc_default_password) }}"
+      timeout: "{{ vsc_command_timeout_seconds }}"
+  when: vsc_creds is not defined
```

Comment:
Using a password here, do you need no_log?

Reply:
Thanks :)